_posts/2021-10-1-updates.md

```diff
@@ -26,24 +26,24 @@ We track the progress on this front on Github:
 
 ## Design work is ongoing
 
-Our lead designer Deanero started to work on Haveno's user interface. Work is ongoing and progresses can be tracker on the Kanban project board on Github: https://github.com/haveno-dex/haveno-design/projects/1. If you wish to contribute to the Design development, join the dedicated room on Matrix (#haveno-design:haveno.network) or IRC/Libera (#haveno-design).
+Our lead designer Deanero started to work on Haveno's user interface. Work is ongoing and progress can be tracked from the Kanban project board on Github: https://github.com/haveno-dex/haveno-design/projects/1. If you wish to contribute to the design development, join the dedicated room on Matrix (#haveno-design:haveno.network) or IRC/Libera (#haveno-design).
 
 ## Bounties
 
-If you wish to work in exchange for XMR take a look at our bounty program. It's already up and running!
+If you wish to work in exchange for XMR, take a look at our bounty program. It's already up and running!
 
 The issues avialable for a bounty in XMR are tracked on a dedicated Kanban board on Github: https://github.com/orgs/haveno-dex/projects/2
 
-Would you like to contribute to Haveno in exchange for XMR but there is nothing that suit you yet? Just contact us and let us know what would you like to work on, if we think it worth it, we will gladly add a bounty for the job. See the [conditions for receiving a bounty](https://github.com/haveno-dex/haveno/blob/master/docs/bounties.md).
+Would you like to contribute to Haveno in exchange for XMR but can't because there is nothing that suits you yet? Just contact us and let us know what would you like to work on- if we think it's worth it, we will gladly add a bounty for the job. See the [conditions for receiving a bounty](https://github.com/haveno-dex/haveno/blob/master/docs/bounties.md) for more information.
```

Comment:
I'm not a big fan of using `-` after words (in this case `work on-`). I would prefer to use a period or a comma.

Reply:
Alright